### DISCUSSION TITLE
Undelegate failed, and got nothing after two month

### DISCUSSION THREAD

**Discussion by zhfnjust** [classic-terra/core#382] [Support]
i undelegate my lunc  two month ago from two validator.

but the transactions failed:

https://finder.terra.money/classic/tx/0303462085E8BC45763CF11E646E29A39B4EFBBD36003BEB3401123AE324C630

https://finder.terra.money/classic/tx/3ED3A5FDE35C43A93CD64FEE20C2598BB3F7EEB5615D12261FDDC742DEE30C55

how to get my coin back?

**Comment by StrathCole** [comment]
https://finder.terra.money/classic/tx/08F8853943B9A9847F5A7F52D4ECF83C86608F3C316FFBEAD9AB3D0B30A19838
https://finder.terra.money/classic/tx/6EE237718D154568B01074679EEB5407693F4C9E1F8AB3C527C5FF44204D9755
https://finder.terra.money/classic/tx/6939986A9AE75FACE5FE950B6CC4B01DA9BF2EFC0DC7FF821B35C33F419C8605

You had also successful txs for the undelegations and the coins are already back.

**Reply by zhfnjust** [reply]
sorry. It's been too long, I remembered it wrong
